### ISSUE TITLE
Closing parentheses appear in the wrong places

### ISSUE THREAD

**Issue by farbodsalimi** [bvieira/sv4git#85]
Both `git sv release-notes` and `git sv changelog` render closing parentheses in the wrong places. I believe this bug shows up when the jira ticket is set. However, we're using the default config for `issue`:

```
    issue:
        regex: '[A-Z]+-[0-9]+'
```

version: v2.8.1

Example:
```sh
git sv release-notes
```

```
## v0.8.0 (2023-01-04)

### Features

- **processor-worker:** add retry policies (#1027) (7362797e)
- **counters-worker:** upgrade aws sdk (#1025) (f0e0908a)
- **counters-worker:** change buffer length (#1021) (4fa49161)

### Bug Fixes

) **query-gateway:** update schema decoding (#1016) (ed06edd1) (TICKET-982

### Misc

) **all:** remove v1 files (#1019) (fc27e4cb) (TICKET-1128

```

**Comment by bvieira** [comment]
Hey, thanks for using sv4git! ❤️

I probably will look at this on this weekend..

can you send me the `git log` of this release, please? seems that there is some new line our trail char that I need to remove.. but at least I can test the behavior here to make sure it's working.. or send me some sample repository with a couple of fake commits so I could have a way to reproduce the bug here.. 

also, are you using the default template or a custom one?

**Comment by farbodsalimi** [comment]
This is a log example:

```
commit 16d998ecf1f480a19cc11c0343addc87849f549a
Author: <ABCD>
Date:   Mon Jan 9 20:14:39 2023 -0700

    fix(query-gateway): history count fix (#1044)

    * fix(query-gateway): history count fix

    parse as string
    comment out tests we think are incorrectly setting timestamp

    jira: TICKET-1008
```

and the output of git-sv:

```
## v1.0.0 (2023-01-11)

### Bug Fixes

) **query-gateway:** history count fix (#1044) (16d998ec) (TICKET-1008
```

After looking at the git logs I think it might be the newlines?!
Does this log example help? or do you still want me to create a dummy repo?

**Comment by bvieira** [comment]
I can try with it.. what is very weird is this parênteses in the beginning of the line 🤔

if the problem is some kind of invisible character, maybe it will be lost after copying and pasting here.. not sure.. if it's not a bother to you, the repo is easier to test.. but I can try and tell you if I'll need it or not

**Comment by farbodsalimi** [comment]
I actually don't see any special characters in my terminal or in github.

**Comment by bvieira** [comment]
hey, I used the same msg that you sent to me in the last comment, but could not reproduce yet

```bash
commit 6afe60bdd82ce63b66e961ac6bcbb6d519186743 (HEAD -> main)
Author: Beatriz Vieira
Date:   Sun Jan 15 00:34:16 2023 -0300

    fix(query-gateway): history count fix (#1044)
    
    * fix(query-gateway): history count fix
    
    parse as string
    comment out tests we think are incorrectly setting timestamp
    
    jira: TICKET-1008
```

and when I run I get:

```bash
$ git sv rn
## v0.0.1 (2023-01-15)

### Bug Fixes

- **query-gateway:** history count fix (#1044) (6afe60b) (TICKET-1008)

```
Which OS are you using? when you do multiline commits do you use any visual editor? I know that sometimes in windows, the new line is represented by `\r\n`, since I remove the `\n`, maybe the problem is the `\n`? :thinking: 

Can you send me the output of `git sv cl`, please? Since it uses a json format, there is a chance that we could spot something there...

```bash
$ git sv cl
{"date":"2023-01-15","timestamp":1673753656,"authorName":"Beatriz Vieira","hash":"6afe60b","message":{"type":"fix","scope":"query-gateway","description":"history count fix (#1044)","body":"* fix(query-gateway): history count fix\n\nparse as string\ncomment out tests we think are incorrectly setting timestamp\n\njira: TICKET-1008","metadata":{"issue":"TICKET-1008"}}}
```

**Comment by farbodsalimi** [comment]
First of all, thank you for debugging this issue so thoroughly.

> Which OS are you using?

MacOS

One thing that I forgot to mention is we see these issues on "squash and merge" a lot. This is another example:

```bash
git sv release-notes                                                                                                                                                      laika/git/main
```

```md
## v1.0.1 (2023-01-15)

### Features

) **all:** v2-ify all inserters (#1040) (255c1f6f) (TICKET-77
```

```json
{"date":"2023-01-12","timestamp":1673546257,"authorName":"GitHub","hash":"255c1f6f","message":{"type":"perf","scope":"all","description":"v2-ify all inserters (#1040)","body":"* perf(all): v2-ify all inserters\r\n\r\ndb-dsn-v2 is used to instantiate the flushers\r\n; it is a dsn.V2 format\r\n\r\njira: TICKET-77\r\n\r\n* fix(all): change config name\r\n\r\njira: TICKET-77\r\n\r\n* test(all): testlib updates\r\n\r\nWriteEvents() is now in db/testlib;\r\nrm GetRandomDB() b/c it's only used in tests\r\nWriteEvents() is now in db/testlib\r\n\r\njira: TICKET-77\r\n\r\n* test: add UT for BuildHosts()\r\n\r\nAlternate hosts seems useless for flusher\r\n\r\njira: TICKET-77\r\n\r\n* chore: add bulk update script\r\n\r\njira: TICKET-77\r\n\r\n* chore: comment out worker in script\r\n\r\njira: TICKET-77","metadata":{"issue":"TICKET-77\r"}}}
```

**Comment by bvieira** [comment]
ohh.. great, so there is a `\r` alone somewhere.. usually the `\r\n` is not a problem, but when we have only a `\r` we can have this weird behavior.. and since it is an invisible character, is not very easy to spot it..

didn´t knew that github added it when squash is used

I'll added a function to remove all `\r` from the commit message, can you use this release and just check if everything is working as expected, please?

[git-sv_dev-202301152344_darwin_amd64.tar.gz](https://github.com/bvieira/sv4git/files/10422110/git-sv_dev-202301152344_darwin_amd64.tar.gz)

**Comment by farbodsalimi** [comment]
Thank you so much! yes the new binary doesn't have that bug!

**Comment by bvieira** [comment]
Greatt 🎉  Thanks a lot for helping me! ❤️ 

Im going to create the tests and prepare the pr...
Also I'll try to fix the other issue to release both together, instead of having 2 releases in a small period of time.. I hope that everything will be ready on this weekend

**Comment by bvieira** [comment]
hey, this feature is available on [v2.9.0](https://github.com/bvieira/sv4git/releases/tag/v2.9.0) :rocket: !!

Thanks again and just let me know if everything is working as expected!!
